src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue

```diff
@@ -27,6 +27,9 @@
 		<div id="app-settings__header">
 			<button class="settings-button"
 				type="button"
+				:aria-expanded="open"
+				aria-controls="app-settings__content"
+				aria-label="t('Open settings menu')"
```

Comment:
Unfortunately, this does not work. It needs to be a computed property, otherwise the l10n extraction does not find the string (seems to be a long standing bug in the script). You can see that the string did not get extracted into the pot file.

Reply:
And you are missing the `:` to make Vue realise the string inside is to be interpreted as JS code:

```suggestion
				:aria-label="t('Open settings menu')"
```

Reply:
Right, thanks! Fixed.

Reply:
As I said, this needs to be a computed property.

Reply:
yeah, both need to be done. Sorry for the confusion